openlibrary/templates/stats/readinglog.html

```diff
@@ -0,0 +1,58 @@
+$def with (stats)
+
+<div id="contentBody">
+  <h1>Reading Log Stats</h1>
+
+  <div class="admin-section">
+    <h2># Books Logged</h2>
+    <p>The total number of books logged using the Reading Log feature</p>
+    <table class="readinglog-stats">
+      <tbody>
+        <tr>
+          <th>All time</th>
+          <td>$stats['total_books_logged']['total']['count']</td>
+        </tr>
+        <tr>
+          <th>This Month</th>
+          <td>$stats['total_books_logged']['month']['count']</td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+
+  <div class="admin-section">
+    <h2># Unique Users Logging Books</h2>
+    <p>The total number of unique users who have logged at least one book using the Reading Log feature</p>
+    <table class="readinglog-stats">
+      <tbody>
+        <tr>
+          <th>All time</th>
+          <td>$stats['total_users_logged']['total']['count']</td>
+        </tr>
+        <tr>
+          <th>This Month</th>
+          <td>$stats['total_users_logged']['month']['count']</td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+
+  <div class="admin-section">
+    <h3>Most Wanted Books (All Time)</h3>
+    <div class="mybooks-list">
+      <ul class="list-books" id="siteSearch">
+        $for book in stats['leaderboard']['most_wanted']: 
+            $:macros.SearchResultsWork(book['work'], decorations="Added %s times" % book['cnt'])
+      </ul>
+    </div>
+
+    <h2>Most Logged Books</h2>
+    <h3>Most Read Books (All Time)</h3>
+    <div class="mybooks-list">
+      <ul class="list-books" id="siteSearch">
+        $for book in stats['leaderboard']['most_read']:
+            $:macros.SearchResultsWork(book['work'], decorations="Added %s times" % book['cnt'])
```

Comment:
If you're going to string interpolate, please get the pluralization right, i.e. "1 times".